mass_editing/migrations/12.0.2.0.2/post-migration.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
```

Comment:
This import will fail because this module does not depend of openupgradelib (for example in out servers such library is not installed).

Is this intentional, @pedrobaeza ?

Reply:
It won't fail, because it's on [the requirements.txt](https://github.com/vauxoo/server-ux/blob/12.0/requirements.txt#L3)

Reply:
@luisg123v This dependecy should be declared into the __manifest__.py as external python dependency.
```python
 "external_dependencies": {"python": ["openupgradelib"]},
```

Reply:
@nhomar first advice: install openupgradelib library everywhere. It's used in a lot of places, and not always for migration scripts, as it has interesting methods for easing certain operations. Second thing: it's the same bad failing due to missing library declared in the manifest than when executing the migration scripts. Doing this script without openupgradelib will lose featues (no logging) or uglier code, so it's reasonable to expect to have it in any deployment of OCA modules. As @luisg123v says, it's on requirements.txt of this repo, and you deploy using it.

Reply:
We have got this debate several times, but if you will block this by this reason, I can add the `external_dependencies` key. I don't add to it because someone installing the module for the first time doesn't need the library.

Reply:
Note that openupgradelib "hidden dependency" was already introduced previously. See #213

Reply:
@pedrobaeza AFAIK, it's just a comment....  IMO it remains important to always declare external dependencies but I sometime also forget to do it or to pay attention to this point in my reviews.

Reply:
But as said, new users don't need such dependency, so I prefer to keep it a it is now, but if you want to add it, I will do it.

Reply:
@pedrobaeza It's ok for me

Reply:
OK, thanks.